.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+exclude: '.*\.(pcap|pcapng|dat|txt)'
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0  # Use the ref you want to point at
+    hooks:
+    -   id: trailing-whitespace
+    -   id: check-case-conflict
+    -   id: end-of-file-fixer
+    -   id: mixed-line-ending
+        args: ['--fix=lf']
```

Comment:
why do we want `lf` and not the default? 🤔

Reply:
Is the most frequent line ending per file?
 
Maybe if this is the case, this will not guarantee a cohenrency in all the repo.

I think we can agree that we want LF as EOL no?

Reply:
I agree on @clementperon, default will not guarantee consistency between files and most of developers work on Linux based systems so I think LF will be more suitable for an open source repo. What do you think?

Reply:
ok sure, I agree 👍